vg/vgsvg/vgsvg.go

```diff
@@ -17,8 +17,10 @@ import (
 	"image/png"
 	"io"
 	"math"
+	"strings"
 
 	svgo "github.com/ajstarks/svgo"
+	stdfnt "golang.org/x/image/font"
 
 	"gonum.org/v1/plot/font"
 	"gonum.org/v1/plot/vg"
```

Comment:
I think somewhere in this file we need documentation about how to ensure that the standard fonts are available on the system if they are not embedded.

Reply:
you mean adding documentation (or a mechanism) to be able to use the `Times/Helvetica/Courier` fonts when embedding isn't enabled?

Reply:
I think just documentation.

Reply:
I've added an example instead.